Add TUtoolbox

chbwien opened the pull request:
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20085)

chbwien committed:
Create TUtoolbox_2021.0.0.332.yaml

New manifest for the TUtoolbox application

chbwien committed:
Merge branch 'master' of https://github.com/chbwien/winget-pkgs

chbwien committed:
Updated manifest

Converted into singleton schema

chbwien committed:
changed folder structure

chbwien committed:
Updated URL to versioned

ItzLevvie commented on manifests/t/TUitTUWien/TUtoolbox/2021.0.0.332/TUitTUWien.TUtoolbox.yaml:
```diff
@@ -0,0 +1,15 @@
+PackageIdentifier: "TUitTUWien.TUtoolbox"
+PackageVersion: "2021.0.0.332"
+PackageLocale: "en-US"
+Publisher: "TU.it, TU Wien"
```
```suggestion
Publisher: "TU Wien - TU.it"
```

chbwien replied:
Changed the , to a -

ItzLevvie commented on manifests/t/TUitTUWien/TUtoolbox/2021.0.0.332/TUitTUWien.TUtoolbox.yaml:
```diff
@@ -0,0 +1,15 @@
+PackageIdentifier: "TUitTUWien.TUtoolbox"
+PackageVersion: "2021.0.0.332"
```
```suggestion
PackageVersion: "TUtoolbox (332)"
```

chbwien replied:
Can not change the version number. we will have a 2021.1.0.x this year and a 2022.0.0.x next year. It's the version of the installer exe and all parts will change.

ItzLevvie replied:
PackageVersion needs to match the one in Control Panel as Windows Package Manager uses this key for `winget list` and `winget upgrade`, otherwise, the PR won't be able to be approved.

chbwien replied:
THX for pointing out the bug in the installer. New installer (333) available, new manifest uploaded for testing.

chbwien committed:
changed manifest as requested

ItzLevvie commented on manifests/t/TUitTUWien/TUtoolbox/2021.0.0.332/TUitTUWien.TUtoolbox.yaml:
```diff
@@ -0,0 +1,14 @@
+﻿PackageIdentifier: "TUitTUWien.TUtoolbox"
+PackageVersion: "TUtoolbox (332)"
+PackageLocale: "en-US"
+Publisher: "TU.it - TU Wien"
```
```suggestion
Publisher: "TU Wien - TU.it"
```

ItzLevvie replied:
Publisher needs to match the one in Control Panel as Windows Package Manager uses this key for `winget list` and `winget upgrade`, otherwise, the PR won't be able to be approved.

ItzLevvie commented on manifests/t/TUitTUWien/TUtoolbox/2021.0.0.332/TUitTUWien.TUtoolbox.yaml:
```diff
@@ -0,0 +1,14 @@
+﻿PackageIdentifier: "TUitTUWien.TUtoolbox"
+PackageVersion: "TUtoolbox (332)"
```
```suggestion
PackageVersion: "TUtoolbox (332)"
```

ItzLevvie reviewed (changes requested):
Please update the version folder as well.

chbwien committed:
Version needs to stay for a versionmanagement. See version of exe.

See other working manifests with full working version numbers
t\Trimble\SketchUpMake\17.2.2555.0
or
Atlassian\Sourcetree\3.4.5.0\Atlassian.Sourcetree.yaml
We will have a 2022.0.0 next year and most likeley a 2021.1.0.xx this year

ItzLevvie reviewed (changes requested):
Please see the latest review.

chbwien committed:
Bug in installer solved. version bumped

chbwien committed:
took publisher from installer
